code/core/src/preview-api/modules/preview-web/parseArgsParam.test.ts

```diff
@@ -58,7 +58,7 @@ describe('parseArgsParam', () => {
   });
 
   it('parses Date with timezone offset', () => {
-    const args = parseArgsParam('key:!date(2001-02-03T04:05:06.789+09:00)');
+    const args = parseArgsParam('key:!date(2001-02-03T04:05:06.789%2B09:00)');
```

Comment:
Are all these updated tests not indications that this is backwards incompatible, and we should delay merging this until the next major version @JReinhold ?

Reply:
I don't agree with this updated format to begin with. `+` is valid in a URL. Moreover, this is fully client-side URL parsing so we don't even have to adhere to the specs necessarily.

Reply:
this instance is what I'm commenting on here:
https://github.com/storybookjs/storybook/pull/28315/files#r1763000236

as for the other tests, it looks to me that they were testing that invalid paths were still "working", and it's fine to me that they don't work the same. I don't see how they could possibly be constructed by us, so the user must have manually written an invalid query param, and I don't feel the need to support that at all.

Reply:
elsewhere, we translate spaces to `+` so maybe I also missed a place where we need to do the same